Exercise_D and E/assets/style/index.css

```diff
@@ -2,3 +2,74 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+header{
```

Comment:
Try to stick to one colour convention, you start with string colour, so use it throughout the stylesheet.